Ik contact us

ihorkharatin opened the pull request:
Оновлена версія для мобілки та планшета

ihorkharatin committed:
mobile-contact-us

ihorkharatin committed:
updated-required-input

ihorkharatin committed:
delete size

ihorkharatin committed:
Merge branch 'main' into ik-contact-us

ihorkharatin committed:
deleted img

ihorkharatin committed:
updated-mob-tablet

updated mobile and tablet

ihorkharatin committed:
Merge branch 'ik-contact-us'

ihorkharatin committed:
updated

ihorkharatin commented:
Добавив коменти і зробив Push origin знову.

ihorkharatin committed:
comments for test

ihorkharatin commented:
Тестую як воно працює

ihorkharatin committed:
new comments

ihorkharatin committed:
Update contact.css

updated desktop version

GYaskey commented on src/partials/contact.html:
```diff
@@ -1,5 +1,102 @@
+          <img
+            src="./img/contact/contact_mob@1x.jpg"
+            alt="Watch"
+             width="" 
```
треба додати базовву ширину і висоту

ihorkharatin replied:
Пробував. Тоді тянеться ця ширина на планшет і десктоп. Я прибрав її.

Gleb-Yas replied:
ти можеш ці розміри задати через стилі в медіа запитах або в кожному source
подивись як це зроблено в hero

ihorkharatin replied:
Добавив

GYaskey commented on src/css/contact.css:
```diff
@@ -0,0 +1,145 @@
+.contact-us {
+  padding-top: 30px;
+  padding-bottom: 30px;
```
паддінги змінюються на планшеті і десктопі

ihorkharatin replied:
Ви правив добавив

GYaskey commented on src/css/contact.css:
```diff
@@ -0,0 +1,145 @@
+  }
+}
+
+.contact-us-container-form {
```
цей окремий контейнер в цілому нічого не задає
всі властивості ідуть від форми
якщо хочеш - можна прибрати

ihorkharatin replied:
Цей контейнер в мене вирівнює колонки і стовпчик. Андрій сьогодні порадив його створити обгорнути форму.

GYaskey commented on src/css/contact.css:
```diff
@@ -0,0 +1,145 @@
+}
+@media only screen and (min-width: 1280px) {
+  .contact-us-form {
+    border-radius: 50px;
```
цей радіус перенесеться з планшета, можна прибрати

ihorkharatin replied:
Я перевіряв на мобілкі 30 радіус на планшеті та компі 50 подивись

GYaskey replied:
все вірно
але ти дублюєш однакове значення на планшеті і десктопі
строка 64 і 69 однакові
69 не потрібна бо радіус 50 перенесеться з планшета

ihorkharatin committed:
testing

testing

GYaskey commented on src/css/contact.css:
```diff
@@ -0,0 +1,147 @@
+  .contact-us-form {
+    border-radius: 50px;
+    padding: 50px 60px;
+    height: 556px;
```
тут висота не має бути визначена
вона має утворюватись від внутрішнього контента
поміряй кожен елемент і задай потрібний розмір там

ihorkharatin replied:
я пробував порізному контейнер получається меньше по висоті за фото. Тому зафіксував. Я все перепробував не знаю де знайти. Якщо я приберу то контейнер уменьшиться. Тут я не знаю де

ihorkharatin replied:
Виправив

GYaskey commented on src/css/contact.css:
```diff
@@ -0,0 +1,147 @@
+  letter-spacing: -0.02em;
+  color: rgba(12, 12, 12, 0.7);
+  margin-bottom: 28px;
+  align-self: flex-start;
```
це не флекс - бокс, на нього не подіє цей параметр

GYaskey replied:
оце не прибрано

ihorkharatin replied:
Виправив на  text-align: start;

GYaskey commented on src/css/contact.css:
```diff
@@ -0,0 +1,147 @@
+@media only screen and (min-width: 768px) {
+  .contact-us-paragraph-text {
+    max-width: 460px;
+    align-self: center;
```
цей теж не подіє
це флексова команда

GYaskey replied:
це теж

ihorkharatin replied:
Виправив

ihorkharatin committed:
textarea

updated height textarea

GYaskey commented on src/css/contact.css:
```diff
@@ -0,0 +1,147 @@
+.contact-us-input {
+  border: 1px solid rgba(12, 12, 12, 0.15);
+  border-radius: 12px;
+  padding: 12px 136px 12px 14px;
```
тут падінг має бути рівномірний з кожної сторони
12px 14px

ihorkharatin replied:
Скопіював що було на макеті. Виправив

GYaskey commented on src/css/contact.css:
```diff
@@ -0,0 +1,147 @@
+  width: 100%;
+  max-height: 42px;
+  outline: transparent;
+  margin-bottom: 18px;
```
в тебе бекграунд білий
по макету має бути прозорий

ihorkharatin replied:
Виправив. Не додивився

GYaskey commented on src/css/contact.css:
```diff
@@ -0,0 +1,147 @@
+  padding: 12px 136px 12px 14px;
+  width: 100%;
+  max-height: 42px;
+  outline: transparent;
```
не заданий стиль тексту
в тебе він сірий, має бути чорний і тд

ihorkharatin replied:
Добавив плейхолдеру колір чорний

GYaskey commented on src/css/contact.css:
```diff
@@ -0,0 +1,147 @@
+@media only screen and (min-width: 768px) {
+  .contact-us-input {
+    width: 100%;
+    align-self: center;
```
теж не спрацює

ihorkharatin replied:
виправив

GYaskey commented on src/css/contact.css:
```diff
@@ -0,0 +1,147 @@
+  border-radius: 12px;
+  outline: transparent;
+  padding-top: 12px;
+  padding-left: 14px;
```
падінги мають бути по всім сторонам

ihorkharatin replied:
Виправив

GYaskey commented on src/css/contact.css:
```diff
@@ -0,0 +1,147 @@
+  padding-top: 12px;
+  padding-left: 14px;
+  width: 100%;
+  height: 100%;
```
висота textarea відрізняється
це впливає на загальний розмір  контейнера
якщо пофіксити ці та інші розбіжності - не треба буде задавати загальну висоту

ihorkharatin replied:
Виправив на мобілкі 84 а десктопі 90

GYaskey commented on src/css/contact.css:
```diff
@@ -0,0 +1,147 @@
+}
+.contact-us-input:focus,
+.contact-us-textarea-input:focus {
+  border: 1px solid #3350e9;
```
ми в діскорді домовились додати анімацію
подивись в каналі notes-and-resourses

ihorkharatin replied:
добавив

GYaskey commented on src/css/contact.css:
```diff
@@ -0,0 +1,147 @@
+
+.contact-us-textarea-input {
+  border: 1px solid rgba(12, 12, 12, 0.15);
+  border-radius: 12px;
```
textarea треба зафіксувати, зараз є можливість його розтягувати
![Screenshot 2024-06-06 at 22 29 50](https://github.com/Gleb-Yas/brainstorm-battalion-team-project/assets/91141358/cf4bd14f-69f9-416b-9376-5e3e7b43faf5)

ihorkharatin replied:
зафіксував

GYaskey commented on src/css/contact.css:
```diff
@@ -0,0 +1,147 @@
+  text-align: center;
+  color: #fbfbfb;
+  border: none;
+  margin-top: 28px;
```
краще прибрати верхній маржин тут і додати нижній маржин на останній лейбл (в якому textarea)

ihorkharatin replied:
Я зробив так тому що на деск топі не діяв падінг від textarea

GYaskey commented on src/css/contact.css:
```diff
@@ -0,0 +1,147 @@
+}
+
+
+.contact-us-image {
```
це все уже задано на загальний тег img
можна прибрати

ihorkharatin replied:
Прибрав

GYaskey commented on src/css/contact.css:
```diff
@@ -0,0 +1,147 @@
+  border: 1px solid #3350e9;
+}
+
+.btn-send {
```
кнопці треба додати ховер і фокус як на макеті з анімацією

ihorkharatin replied:
Добавив

GYaskey replied:
у кнопки теж має бути анімація

ihorkharatin committed:
Update notifications

ihorkharatin committed:
Updated notices

ihorkharatin commented:
Все виправив лише

1 Якщо фото задати фіксовану ширину вона тянеться на десктоп і планшет
2 Форму поки фіксую якщо прибрати то вона меньша за фото. Поки не знаю попробую ще

ihorkharatin committed:
img

@media for img

ihorkharatin committed:
input form

Знайшов input 42 px for mobile 52 px for desktop
Now the size of the form according to the content  and not fixed.

ihorkharatin committed:
img width

img width fixed in html

ihorkharatin committed:
Merge branch 'main' of https://github.com/Gleb-Yas/brainstorm-battalion-team-project

ihorkharatin committed:
Merge branch 'main' into ik-contact-us

ihorkharatin committed:
Merge branch 'main' of https://github.com/Gleb-Yas/brainstorm-battalion-team-project

ihorkharatin committed:
Merge branch 'main' into ik-contact-us

ihorkharatin committed:
updated

Виправив зауваження

ihorkharatin committed:
Merge branch 'main' of https://github.com/Gleb-Yas/brainstorm-battalion-team-project

ihorkharatin committed:
Merge branch 'main' into ik-contact-us

ihorkharatin committed:
border radius

ihorkharatin committed:
placeholder color
